### ISSUE TITLE
Binary provided for macos in release section is not working

### ISSUE THREAD

**Issue by itsdivyanshjain** [guardianproject/orbot-apple#91]
Binary provided for macos `Orbot.pkg` in release section is not working. Showing error `Application "ORBOT" can't be opened`. Also, I believe build guide is outdated too in `README` section.

**Comment by tladesignz** [comment]
> Binary provided for macos `Orbot.pkg` in release section is not working. Showing error `Application "ORBOT" can't be opened`. 

Yeah, sorry. This is more or less provided "just because". You will need to re-sign with your own credentials.

To be able to distribute this outside the app store, we would need to change the app, so it doesn't provide a "Network Extension", but a "System Extension".

I spent some time on it, already, but to change this is non-trivial, especially in a project which also contains an iOS app using a "Network Extension". There are all sorts of stupid rules and different signing mechanism around it.

Since the macOS version is just a side-effect of the iOS app and a means for easier debugging, there is currently zero money available to make this happen.

> Also, I believe build guide is outdated too in `README` section.

In what way? Well, the minimal OS and Xcode version is probably slightly outdated. But else?

**Comment by itsdivyanshjain** [comment]
Need a bit short guide/resources about how to make this work, if i want to sideload this app on macos. 

>> Also, I believe build guide is outdated too in README section.

> In what way? Well, the minimal OS and Xcode version is probably slightly outdated. But else?

In git command repo contains different name `orbot-ios` instead of `orbot-apple` that's why felt it is outdated, also when I try to build even after resolving this it failed, don't know the exact error.

**Comment by tladesignz** [comment]
Oh, ha! Good catch.

When you provide an error log, I might be able to help. Otherwise: 🤷

**Comment by itsdivyanshjain** [comment]
> Need a bit short guide/resources about how to make this work, if i want to sideload this app on macos.

And what about this?

**Comment by tladesignz** [comment]
> > Need a bit short guide/resources about how to make this work, if i want to sideload this app on macos.
> 
> And what about this?

Sorry, overlooked that.
Can't help you, there, sorry. Never did it myself. This pkg is seriously provided just because we can. No instructions come along with it.

If you don't want to install from the macOS app store, don't have a clue how to resign an app pkg yourself, then your best bet is to build from scratch.

For the latter, I might be able to help you, if you run into specific problems, but I'm also not an Xcode beginner's tutor.

There's always another option:
You can still just run a plain Tor yourself on macOS. E.g. install via Homebrew. It's a command line tool, which makes it necessary to read some docs, but it has the full power.

And if your whole intention is to do web browsing, anyway, then please just use Tor Browser. Using Tor with another browser can be dangerous, if you don't know, what you're getting yourself into.

**Comment by itsdivyanshjain** [comment]
>If you don't want to install from the macOS app store, don't have a clue how to resign an app pkg yourself, then your best bet is to build from scratch.
For the latter, I might be able to help you, if you run into specific problems, but I'm also not an Xcode beginner's tutor.

Thanks for the update, but I will go with this route when no other option left.

>There's always another option:
You can still just run a plain Tor yourself on macOS. E.g. install via Homebrew. It's a command line tool, which makes it necessary to read some docs, but it has the full power.

Basically i want to run TOR as a vpn connection, that's why i chose to go with ORBOT. There is another solution that you mentioned which i already tried with homebrew to install TOR & use it as SOCKS proxy. But i want to use it as vpn connection, if you can help me in this I really appreciate that.

>And if your whole intention is to do web browsing, anyway, then please just use Tor Browser. Using Tor with another browser can be dangerous, if you don't know, what you're getting yourself into.

Currently, using tor as testing purposes on VM, so I'm fine with risk that comes along with

**Comment by tladesignz** [comment]
I guess, Orbot macOS is actually your best choice, then.
